routs/events.js

```diff
@@ -0,0 +1,122 @@
+const fs = require('fs');
+const path = require('path');
+const express = require('express');
+const csv = require('csvtojson');
+const { Parser } = require('json2csv');
+const { EVENTSDTOPATH } = require(path.join(__dirname, '..', 'config.js'));
+const { log } = require(path.join(__dirname, '..', 'logger.js'));
+
+const router = express.Router();
+
+router.get('/', async (req, res) => {
+  const defaultSearchParams = ['id', 'title', 'location', 'date', 'hour'];
+  const queryParams = Object.keys(req.query);
+  const actualSerchParams = queryParams.filter(p => defaultSearchParams.includes(p));
+  const events = await csv().fromFile(EVENTSDTOPATH);
```

Comment:
we do not have `try/catch` block, so if smth happens we won't handle the error properly for ours request.